src/openeo_aggregator/backend.py

```diff
@@ -500,21 +527,23 @@ def get_process_registry(self, api_version: Union[str, ComparableVersion]) -> Pr
             # TODO: more relaxed version check? How useful is this check anyway?
             _log.warning(f"API mismatch: requested {api_version} outside of {self.backends.get_api_versions()}")
 
-        combined_processes = self.get_merged_process_metadata()
-        process_registry = ProcessRegistry()
+        combined_processes, extra = self.get_merged_process_metadata()
+        process_registry = AggregatorProcessRegistry(federation_missing=extra.get("federation_missing", []))
         for pid, spec in combined_processes.items():
             process_registry.add_spec(spec=spec)
         return process_registry
 
-    def get_merged_process_metadata(self) -> Dict[str, dict]:
+    def get_merged_process_metadata(self) -> Tuple[dict, dict]:
         return self._memoizer.get_or_call(
             key=("all", str(self.backends.api_version_minimum)),
             callback=self._get_merged_process_metadata,
         )
 
-    def _get_merged_process_metadata(self) -> dict:
+    def _get_merged_process_metadata(self) -> Tuple[dict, dict]:
```

Comment:
This change trips up some tests about caching, let's stick to `dict` return here